changes/tasks.py

```diff
@@ -128,6 +128,7 @@ def change_messaging(self, change):
         current_rate = len(current_days.split(','))  # msgs per week
 
         # Deactivate subscriptions
+        utils.deactivate_subscription(current_sub)
```

Comment:
I don't really understand this addition. I mean it seems perfectly correct but I don't understand how it was working before? :)

Reply:
Ah sorry I do understand now. Because we're pulling the first element of the generator we need to process that one separately to the ones below 👍